h1b/src/hil/flash/h1b_hw.rs

```diff
@@ -0,0 +1,249 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use kernel::common::cells::VolatileCell;
+
+#[repr(C)]
+pub struct H1bHw {
```

Comment:
You should probably use the register crate here, e.g. as in 

https://github.com/google/tock-on-titan/blob/master/h1b/src/usb/registers.rs

It's probably fine to leave unused registers as VolatileCell<u32>s; entering lots of register definitions that aren't tested is more bug-prone than leaving unused registers for future people to enter and test.

Reply:
I switched to the register crate where I think it made sense.